Formula/zxing-cpp.rb

```diff
@@ -0,0 +1,38 @@
+class ZxingCpp < Formula
+  desc "C++ port of the ZXing barcode decoder"
+  homepage "https://github.com/zxing/zxing"
+  url "https://github.com/zxing/zxing/releases/download/zxing-2.3.0/ZXing-2.3.0.zip"
+  sha256 "a9b9c7e2cd61c7918dc5f8856c0f91665ab6a109c6cd30ce1795d09bf4baf9c4"
+  head "https://github.com/glassechidna/zxing-cpp.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    if build.stable?
+      cd "cpp"
+    end
+
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+
+      if build.stable?
```

Comment:
I don't think that would work, given that [the C++ port was deleted in 2013](https://github.com/zxing/zxing/commit/4856e86a46a5836d7ea793a78bea5637c85fe7c6#diff-ac731f2fedd74089d1149d83620eeb92) from the main repo. [zxing-cpp](https://github.com/glassechidna/zxing-cpp) forked that code, and is the unofficial-official active C++ zxing repo (but with no releases, currently).

I know it's a sorry state of affairs, but it's one that is common to barcode/QR decoding libraries, and unfortunately what C++ developers have to work with for the time being.